### ISSUE TITLE
tocSummary and headings settings not working together

### ISSUE THREAD

**Issue by simondean** [Mermade/widdershins#121]
Hi.  I'm seeing a weird issue when I use the `tocSummary` and `headings` settings together.  I've created the following example:

widdershins config: https://github.com/simondean/widdershins-issues/blob/master/headings-depth-with-toc-summary/widdershins.yaml

```yaml
resolve: true
tocSummary: true
headings: 3
```

Example HTML: https://simondean.github.io/widdershins-issues/headings-depth-with-toc-summary/#Example-API

## Actual TOC

In my `widdershins.yaml` file I've set `tocSummary` to `true` and `headings` to `3`.  I expect to get three levels of TOC but what I'm actually getting is a single level of TOC.  Which is weird, as the default levels of TOC is 2, not 1.  

```
Example API v1.0.0
Endpoints
```

See https://simondean.github.io/widdershins-issues/headings-depth-with-toc-summary/#Example-API to see the HTML that's being generated.  

## Expected TOC

```
Example API v1.0.0
    Example H2
        Example H3
Endpoints
    getGreeting
```

Thanks

**Comment by simondean** [comment]
I’m using Widdershins with Shins here.

**Comment by MikeRalphson** [comment]
A fix for the `--tocSummary` option is included in `v3.2.7`. It aims to prevent the Slate ToC javascript from crashing.

**Comment by simondean** [comment]
Hi @MikeRalphson.  I upgraded to Widdershins to `3.4.0` and that fixed it.  Thanks!
